app/scripts/modules/core/src/domain/ISecurityGroup.ts

```diff
@@ -14,21 +16,28 @@ export interface IUsages {
   serverGroups: IServerGroupUsage[];
 }
 
-export interface ISecurityGroup {
+export interface ISecurityGroup extends IIndexed {
   account?: string;
   accountId?: string;
   accountName?: string;
   application?: string;
   cloudProvider?: string;
+  cluster?: string;
```

Comment:
What's the `cluster` field on a security group? Same with `sequence`, `serverGroup`...

Reply:
I'm not sure where that came from, it's not even on the model in the search results.  Probably an erroneous addition.  It's removed by reverting the domain object changes and adding new ones specific to search results.